src/io/sarnowski/swagger1st/schemas/swagger_2_0.clj

```diff
@@ -1,5 +1,6 @@
 (ns io.sarnowski.swagger1st.schemas.swagger-2-0
-  (:require [schema.core :as s]))
+  (:require [clojure.string]
```

Comment:
I have no idea what "more explicit things" means here but I've changed the require to be consistent.